README.md

```diff
@@ -19,7 +19,7 @@
 
 ![DataHub](docs/imgs/datahub-logo.png)
 
-> 📣 Next DataHub town hall meeting on Oct 30th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/)) 
+> 📣 Next DataHub town hall meeting on November 06th, 9am-10am PDT ([convert to your local time](https://greenwichmeantime.com/time/to/pacific-local/))
```

Comment:
Just "6th", no leading 0. Ditto below.

American thing? I've not seen "Month Day, Year" where day has a leading zero before.

Reply:
I have been adding 0s all the way till now  😄